tensorflow_addons/optimizers/moving_average_test.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for MovingAverage optimizers."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import six
+
+import tensorflow as tf
+
+import moving_average
+from tensorflow_addons.utils import test_utils
+
+
+class MovingAverageTest(tf.test.TestCase):
+    @test_utils.run_deprecated_v1
+    def test_run(self):
```

Comment:
Could you add a test running in eager mode? (or make tests compatible with both graph and eager modes)
Like patterns here:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/optimizer_v2/adam_test.py#L281

Reply:
That's what I did initially. But `tf.train.ExponentialMovingAverage` doesn't seem to work in eager mode.